src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java

```diff
@@ -33,8 +33,9 @@ public static void main(String[] args) {
      * @param milliseconds время (в миллисекундах) между срабатыванием датчика света и звука
      */
     static float getDistance(float milliseconds) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+        float speed = (float)(SOUND_SPEED / 3.6);
+        float time = milliseconds/1000;
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"